site/content/docs/5.2/forms/layout.md

```diff
@@ -119,6 +119,13 @@ Create horizontal forms with the grid by adding the `.row` class to form groups
 
 At times, you maybe need to use margin or padding utilities to create that perfect alignment you need. For example, we've removed the `padding-top` on our stacked radio inputs label to better align the text baseline.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+This form variant, with **labels** that are **not above input fields**, should not be used because it does not respect the Orange Design System specifications.
```

Comment:
Everywhere that we talk about "with **labels** that are **not above input fields**," should be replaced by the notion of "horizontal form" or "horizontal layout". I don't know how to do that but we should only explain that somewhere and not having danger callouts everywhere but rather having a warning callout referencing this rule. Most of the time, we have danger callouts while the concept is OK.

Reply:
OK

Reply:
OK

Reply:
Resolved ?